### DISCUSSION TITLE
Desktop download links and explanation of different .bat

### DISCUSSION THREAD

**Discussion by ppete2** [devemux86/cruiser#240] [Q&A] [answered]
Hi,
For users which aren't long-time Cruiser users, no computer or Github experts: please add the following things to the Wiki-page:

- the exact download links for Windows/macOS/Linux versions, e.g. https://github.com/devemux86/cruiser/releases

- a short description when to use and whats the difference of the different .bat/.sh files:
  _cruiser.bat, cruiser-mapsforge.bat, cruiser-vtm.bat_. And if there's a disadvantage to start Cruiser by clicking onto _"cruiser.jar"_

**Comment by devemux86** [comment, marked as answer]
@ppete2 thanks for the suggestion.

The installation instructions of desktop Cruiser for each platform (Windows, macOS, Linux) are in the [OpenStreetMap Wiki](https://wiki.openstreetmap.org/wiki/Cruiser).
The Wiki is also mentioned in the GitHub main page. I added the desktop download link in the Wiki installation instructions.

The executable files with the "-mapsforge" or "-vtm" open Cruiser directly with the specific map engine.
Usually you can follow the installation guide and just run the recommended "cruiser.bat" or "cruiser.sh".

**Comment by ppete2** [comment]
Maybe a little background why I made these suggestions, in the view of a Cruiser-newbie:

I'm not a computer newbie, and also already had worked with differnt Github repos several times before. But although I did my best, I didn't know that I had to click onto the somewhat hidden link "Releases" somewhere on the right side of the Github page. I even thought that cruiser is only for mobile phones although Windows etc. are mentioned on the Wikisite, since the just the links to Google play are prominetatly visible. So it would make things a lot user-friendlier if you would post the direct link to this Release - not just the overall Github-site on the Wiki-site.

Second, the different .bat files and "cruiser.jar": I wondered why with all .bat files and the .jar-file Cruiser opens with no visual difference. I'm also using other Java-based software which I start by clicking onto its .jar file. So I wondered if theres' a disadvantage by starting Cruiser in this way? Some short lines on the Wiki would make the differences clear for interested users.

**Reply by devemux86** [reply]
Jar files are not executable (depending the operating system), they often open with the archive manager.

.bat and .sh files are useful to pass more options, like extra JVM memory or extra parameters to the app.

**Reply by ppete2** [reply]
At least on my Windows-10 PC they are. So I wonder why? ;-)
But as suggested these answers are better put onto the Wiki - thats why a Wiki is for

**Comment by Cer-o** [comment]
Hello, I am a user with little experience in slightly more complex issues.
I have read something like Cruiser Desktop, does that mean that there is a program to be able to have Cruiser on my computer?
On my computer I have Linux Mint.
Thanks and sorry for this stupid question.

**Reply by devemux86** [reply]
@Cer-o I work on Linux! :slightly_smiling_face: 

Desktop Cruiser is a Java app that can be used on desktop computers (Windows, macOS, Linux).
Please see the installation instructions here: https://wiki.openstreetmap.org/wiki/Cruiser#Installation

I created desktop Cruiser many years ago to help offline map and theme community with their tests.

**Reply by 0709wiwiwi** [reply]
@Cer-o 
> there is a program to be able to have Cruiser on my computer?

I do use Windows and yes, sure you can, and the desktop display is big and clean.    
I do start the Cruiser program by using the .bat file with a shortcut to the desktop. 
You can than attach the provided Cruiser **[C]**  .ico image to display the shortcut. 

One negative when using the Cruiser desktopversion  is that the offline Brouter is not available !

Put your .kurviger or gpx navtrack files in  a"Tracks" Folder and share the folder onto Dropbox (free) or Google drive (free).
For example by the Android "Cx File explorer" network connection explore your Dropbox or Google drive content.

Most optimal  is auto synchronisation between the desktop and an identical Android  "Tracks" folder.  
The automatic synchronisation can be handled by the next Android apps.
For Dropbox use Dropsync  &  For Google Drive use Drivesync.  Both apps are similar.

**Reply by devemux86** [reply]
> One negative when using the Cruiser desktopversion is that the offline Brouter is not available !

On desktop it is often easier to use the routing websites.

I had discussed in the past with @afischerdev what is needed to implement BRouter (offline) on desktop.
It requires significant work, while the [BRouter website](https://brouter.de/brouter-web/) already offers extensive options in the browser.

Also my available resources and focus is on Android Cruiser. :slightly_smiling_face:

**Reply by Cer-o** [reply]
Muchas gracias, pero veo que es mucho más complicado de lo que mis conocimientos pueden abordar.

**Reply by 0709wiwiwi** [reply]
@Cer-o 

[Cruiser Desktop](https://youtu.be/GEGl9iAmlPs)

If you see the clean Cruiser desktop presentation you might still be tempted ?
